requirements-dev.txt

```diff
@@ -3,3 +3,4 @@ pytest-django
 pytest-cov
 pytest-xdist
 flake8
+django-libsass
```

Comment:
You can remove it from the requirements-dev.txt list, that's installed on top of what's already installed in `requirements.txt`.